packages/bech32m/src/passet.ts

```diff
@@ -0,0 +1,24 @@
+import { fromBech32m, toBech32m } from './format/convert';
+import { Inner } from './format/inner';
+import { Prefixes } from './format/prefix';
+
+const innerName = Inner.passet;
+const prefix = Prefixes.passet;
+
+export const bech32mAssetId = ({ [innerName]: bytes }: { [innerName]: Uint8Array }) =>
+  toBech32m(bytes, prefix);
+
+export const assetIdFromBech32m = (passet1: string): { [innerName]: Uint8Array } => ({
+  [innerName]: fromBech32m(passet1 as `${typeof prefix}1${string}`, prefix),
+});
```

Comment:
The whole purpose of this package is to allow us easy ways to convert to/from bech32 to/from Penumbra protobuf types. Returning approximations of the inside of these types feels unnecessary, especially given all consumers will then be required to construct the type manually right after.  The more natural API is to simply return what they need. I think we should do this:

```diff
- export const assetIdFromBech32m = (passet1: string): { [innerName]: Uint8Array }
+ export const assetIdFromBech32m = (passet1: string): AssetId
```

Reply:
the whole purpose of the package is to convert from bech32m to binary data in a verifiable way.

the types are a convenience. i would prefer to remove the structure than to add the types.